### DISCUSSION TITLE
[UNet2DConditionModel] I don't want to perform conditional embedding on the UNet network

### DISCUSSION THREAD

**Discussion by island443** [huggingface/diffusers#7361] [General]
# predict the noise residual
model_pred = unet(noisy_latents, timesteps, encoder_hidden_states).sample

 I don't want to perform conditional embedding on the UNet network,It means not passing in the encoder_hidden_state parameter when predicting noise.What should I do? Do I need to modify the [UNet2DConditionModel] source code?

**Comment by tolgacangoz** [comment]
Does [`UNet2DModel`](https://github.com/huggingface/diffusers/blob/25caf24ef90fc44074f4fd3712f6ed5a1db4a5c3/src/diffusers/models/unets/unet_2d.py#L40) help?

**Reply by island443** [reply]
Thank you for your answer,  I see that pre trained stable diffusion models are all using UNet2DConditionModel instead of UNet2DModel. Can I use UNet2DModel for latents space?
This seems to require retraining rather than fine-tuning？

**Reply by tolgacangoz** [reply]
If Stable Diffusion models' `UNet2DConditionModel` have trained with `encoder_hidden_states` parameter, then I think we have to use it, otherwise it would be another network. Does [`LDMPipeline`](https://huggingface.co/docs/diffusers/main/en/api/pipelines/latent_diffusion_uncond) help?

**Reply by island443** [reply]
LDMPipeline has been very helpful to me. Thank you very much again and wish you every success。
